nc-games-fe/src/Components/utils.js

```diff
@@ -28,3 +28,9 @@ export function fetchCategories() {
     })
 }
 
+export function updateReviewVotes(reviewId){
+    return gamesApi.patch(`/reviews/${reviewId}`, {inc_votes: 1})
+    .then((res) => {
+        return res.data;
+    })
```

Comment:
Good to see an separate api call for this, function is well named. Good to see you've hard coded a +1 for the upvote. This works for now, have a think about if you wanted a 'downvote' button in the future, how would you amend this? Not critical, and no need to implement for this PR.